src/libs/actions/Device/getDeviceInfo/getOSAndName/types.ts

```diff
@@ -0,0 +1,4 @@
+// to keep same behavior from before migration
```

Comment:
```suggestion
// To keep same behavior from before migration
```

What behavior are we talking about?